draft-opsarea-rfc5706bis.md

```diff
@@ -362,6 +351,23 @@ author:
       discussion of data models can be found in {{?RFC3444}}, {{sec-interop}},
       and {{sec-mgmt-info}}.
 
+   *  Fault: See {{?I-D.ietf-nmop-terminology}}.
+
+   *  Fault management: Fault management is the process of interpreting fault notifications and other alerts
+      and alarms, isolating faults, correlating them, and decucing underlying
+      causes. See {{sec-fm-mgmt}} for more information.
+
+   *  Information Model: An abstraction and representation of the
+      entities in a managed environment, their properties, attributes
+      and operations, and the way that they relate to each other. The model is
+      independent of any specific repository, software usage, protocol,
```

Comment:
```suggestion
      independent of any specific software usage, protocol,
```

Reply:
I'm happy to remove 'repository', but any specific reason?

Reply:
I mean, beyond the above -- the term is coming directly from RFC 5706 and not introduced by this effort.